.travis.yml

```diff
@@ -15,14 +15,13 @@ before_install:
 - export DISPLAY=:99.0
 - sh -e /etc/init.d/xvfb start
 
-# Build a production build of the project
-before_script:
-  - ng build --prod --build-optimizer
+matrix:
+  include:
+  - before_script:
+    - ng build --prod --build-optimizer
 
-# Check for coverage, and linting
-script:
-  - ng test --single-run --code-coverage --reporters=coverage-istanbul
-  - ng lint
```

Comment:
Since ```ng test --single-run --code-coverage --reporters=coverage-istanbul``` and ```ng lint``` are independent commands, you can create one more thread to optimize travis build. Create one more ```- script``` and add ```ng test --single-run --code-coverage --reporters=coverage-istanbul``` under it. similar to ```ng lint```. I guess it should work please check.

Reply:
please look `ng test --single-run --code-coverage --reporters=coverage-istanbul` is already included under `yarn test` running it again on different thread is useless.

Reply:
Okay, thanks for the info.